packages/web/app/views/UserProfile/UserProfileView.jsx

```
@@ -0,0 +1,291 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { Box, Typography, Button, Card, CardContent, CircularProgress } from '@material-ui/core';
+import { Person, Lock, Error } from '@material-ui/icons';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)